web/src/unread.ts

```diff
@@ -849,51 +972,59 @@ export function declare_bankruptcy() {
     unread_mention_topics.clear();
 }
 
-export function get_unread_pm(include_per_bucket_latest_msg_id = false) {
+export function get_unread_pm(include_per_bucket_latest_msg_id = false): DirectMessageCountInfo {
     return unread_direct_message_counter.get_counts(include_per_bucket_latest_msg_id);
 }
 
-export function get_unread_topics(include_per_topic_latest_msg_id = false) {
+export function get_unread_topics(include_per_topic_latest_msg_id = false): UnreadTopicCounts {
     const include_per_topic_count = true;
     return unread_topic_counter.get_counts(
         include_per_topic_count,
         include_per_topic_latest_msg_id,
     );
 }
 
-export function get_counts() {
-    const res = {};
-
-    // Return a data structure with various counts.  This function should be
-    // pretty cheap, even if you don't care about all the counts, and you
-    // should strive to keep it free of side effects on globals or DOM.
-    res.direct_message_count = 0;
-    res.mentioned_message_count = unread_mentions_counter.size;
-    res.direct_message_with_mention_count = direct_message_with_mention_count.size;
-
-    // This sets stream_count, topic_count, and home_unread_messages
+type VariousCounts = {
```

Comment:
Probably `FullUnreadCountsData` would be a good name for this.

Reply:
Changed to that, thanks!